README.md

```diff
@@ -59,23 +59,23 @@ Now you will see a review with linting errors...
     ```
 - [gitleaks](https://github.com/zricethezav/gitleaks)
     ```
-    $ gitleaks --path=. -q | lintly --format=gitleaks
+    $ gitleaks --path=. --redact --no-git --report=>(lintly --format=gitleaks)
```

Comment:
Got this tip from here: https://stackoverflow.com/questions/7756609/pass-stdout-as-file-name-for-command-line-util

Went with the [process substitution](https://en.wikipedia.org/wiki/Process_substitution) solution, which I think is the cleanest for this use case.

Reply:
This is a really cool way of passing output. My suggestion here is let's keep 1st option as it is and add `$ gitleaks --path=. --redact --no-git --report=>(lintly --format=gitleaks)`
with additional notes. So that we can be consistent  with the formats for other tools. But it is upto you.

Reply:
Well that's the problem -- the first option just plain *doesn't work*. `gitleaks -q` does *not* output valid JSON (it's more of a 'logging' one single json object at a time to stdout). `lintly` would break.

Given the choice, I would always prefer the more consistent option. I really hate how this one looks but that's what's needed to get it working.

I guess the other option is to do something like
```
gitleaks --path=. --redact --no-git --report=>(tee) | lintly --format=gitleaks
```
but we're making an extra unneeded call to `tee` *just* to make it look consistent. (I'm also not 100% sure this works right here.)